writer.go

```diff
@@ -248,6 +253,9 @@ func NewWriter(config WriterConfig) *Writer {
 	if config.RebalanceInterval == 0 {
 		config.RebalanceInterval = 15 * time.Second
 	}
+	if config.ConnMaxIdleTimeout == 0 {
+		config.ConnMaxIdleTimeout = 540 * time.Second
```

Comment:
```suggestion
		config.ConnMaxIdleTimeout = 9 * time.Minute
```
This would be simpler to read for maintainers?